linen_examples/cloud/launch_gce.py

```diff
@@ -0,0 +1,174 @@
+# Script for creating VM on Google cloud and run a Flax example inside.
+# See ./README.md for instructions.
+
+import datetime
+import os
+import re
+import subprocess
+
+from absl import app
+from absl import flags
+from absl import logging
+
+# General options.
+flags.DEFINE_bool(
+    'dry_run',
+    False,
+    help='If set, then the command to launch the GCE instance will only be '
+    'printed to stdout.')
+
+# Machine configuration.
+flags.DEFINE_string('project', None, help='Name of the Google Cloud project.')
+flags.DEFINE_string('zone', None, help='Zone in which the VM will be created.')
+flags.DEFINE_string(
+    'machine_type', None, help='Zone in which the VM will be created.')
+flags.DEFINE_string(
```

Comment:
Maybe add examples for what this flag can be? Like, is it `"v100"`?

Reply:
Good point. Added command to list these.